src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -14,6 +14,7 @@ internal interface IOktaApi {
 	Task<OktaAuthenticateState> AuthenticateOktaAsync( AuthenticateOptions authOptions );
 	Task<OktaAuthenticatedState> AuthenticateChallengeMfaOktaAsync( OktaAuthenticateState state, int selectedMfaIndex,
 		string challengeResponse );
+	Task IssueSmsChallengeMfaOktaAsync( OktaAuthenticateState state, int selectedMfaIndex );
```

Comment:
For clarity with what this is in Okta (https://developer.okta.com/docs/reference/api/factors/#factors-that-require-a-challenge-and-verify-operation).

Lets rename it to something that reflects that its issuing an MFA challenge, since its not just SMS that follows this flow.

Reply:
IssueChallengeMfaOktaAsync?

Reply:
IssuMfaChallengeOktaAsync rolls  off the tongue a wee bit better :P . 

We should at some point drop the word Okta from the interface methods, it seems redundant when the interface has Okta in the name.

Reply:
Oh true

Reply:
I can create a PR to do that once this change is in

Reply:
Yep, I meant that as an aside, no need to block this PR